src/middlewares/__tests__/validator.js

```diff
@@ -0,0 +1,130 @@
+const middy = require('../../middy')
+const jsonBodyParser = require('../jsonBodyParser')
+const validator = require('../validator')
+
+describe('📦  Middleware Validator', () => {
+  test('It should validate an incoming object', () => {
+    const handler = middy((event, context, cb) => {
+      cb(null, event.body) // propagates the body as a response
+    })
+
+    const schema = {
+      required: ['headers', 'body'],
```

Comment:
Are we sure `body` is mandatory? it might be stripped in GET requests (as it happens with `pathParameters` and `queryStringParameters` when they are not available)